docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller_test.go

```diff
@@ -102,8 +101,8 @@ var _ = Describe("CronJob controller", func() {
 				After creating this CronJob, let's check that the CronJob's Spec fields match what we passed in.
 				Note that, because the k8s apiserver may not have finished creating a CronJob after our `Create()` call from earlier, we will use Gomega’s Eventually() testing function instead of Expect() to give the apiserver an opportunity to finish creating our CronJob.
 
-				`Eventually()` will repeatedly run the function provided as an argument every interval seconds until
-				(a) the function’s output matches what’s expected in the subsequent `Should()` call, or
+    `Eventually()` will repeatedly run the function provided as an argument every interval seconds until
```

Comment:
Space in the const for we to have the same result.

Reply:
It sometimes shows to have the same space but has not.
What I do is copy and past from another line that is working and adding on front to ensure that has the same.